components/markets/MarketAssetDetails.tsx

```diff
@@ -129,6 +129,7 @@ const MarketAssetDetails = observer(
                     )
                   : null,
               change: priceChange,
+              poolId: market.pool.poolId,
```

Comment:
Is this needed?

Reply:
Accidentally popped some stashed code here. Will remove.